core/server/auth/oauth.js

```diff
@@ -86,6 +86,10 @@ function exchangeAuthorizationCode(req, res, next) {
             }));
         }
 
+        if (req.brute && req.brute.reset) {
+            req.brute.reset();
```

Comment:
> As for the exchangeRefreshToken, this may just be my misunderstanding, I felt that exchangeAuthorizationCode would also be called during the token exchange -if not then I certainly need to add a reset call here :)

Authorization codes are only exchanged during initial authentication - the auth code is what the client gets from the OAuth server via the popup and is then used by the ghost server to contact the OAuth server and exchange it for a ghost<->OAuth server access token. This should happen rarely.

Once a client<->ghost session is established that's where the `exchangeRefreshToken` comes into play as the client will make a request at least once an hour to get a new `access_token` - this is where I saw the 429 response.

Hopefully that helps in configuring request limits and resets?